rollup.config.js

```diff
@@ -16,7 +15,9 @@ if (pkgInfo.peerDependencies)
 if (pkgInfo.dependencies) external.push(...Object.keys(pkgInfo.dependencies));
 
 // TODO: query-string does not have an ESM build, so we'll just bundle and ship it
-external = external.filter(x => x !== 'query-string');
+external = external.filter(
+  x => x !== 'query-string' && x !== 'styled-components'
```

Comment:
I'm bundling styled-components as the `package.json:module` field points towards an ESM version compatible with bundlers but **not** browser-ready - `packge.json:browser` points towards what should be a browser-ready ES module version of styled-components but even this includes some non-ES compat dependencies (like `stylis-rule-sheet`) and given that unpkg does not understand this `package.json` field, at the moment it is easier to just bundle styled-components.
Note that this could lead to a stale dependency and we should try and find a workaround (but this is mostly a responsibility of the `styled-components` team).